source/extensions/filters/common/local_ratelimit/local_ratelimit.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
```

Comment:
IMO you can just remove this interface, it's just included as a helper/mixin where it's used so I think it can just expose `requestAllowed()` as a public function on the class?

Reply:
yeah I agree.